apps/backend/src/lib/controller/chip/managed/leaderboard/index.ts

```diff
@@ -1,6 +1,33 @@
-import { ChipIssuer, errorToString } from "@types";
+import {
+  ChipIssuer,
+  errorToString,
+  LeaderboardEntry,
+  LeaderboardEntrySchema,
+  LeaderboardPosition,
+  LeaderboardPositionSchema
+} from "@types";
 import { ManagedChipClient } from "../client";
 
+ManagedChipClient.prototype.GetLeaderboardEntry = async function (
```

Comment:
`GetLeaderboardEntry` may not be used by the frontend. If it isn't I'm inclined to keep it -- it was useful for testing.